user/yaml/README.md

```diff
@@ -0,0 +1,327 @@
+# Deploy **KubeStellar** service in a cluster
+
+Table of contests:
+- [Deploy **KubeStellar** service in a cluster](#deploy-kubestellar-service-in-a-cluster)
+  - [Deploy **KubeStellar** in a **Kubernetes** cluster (**Kind** cluster)](#deploy-kubestellar-in-a-kubernetes-cluster-kind-cluster)
+  - [Deploy **KubeStellar** in an **OpenShift** cluster](#deploy-kubestellar-in-an-openshift-cluster)
+  - [Access **KubeStellar** service directly from the host OS without `admin.kubeconfig` or plugins](#access-kubestellar-service-directly-from-the-host-os-without-adminkubeconfig-or-plugins)
+  - [Access **KubeStellar** service from the host OS by extracting the `admin.kubeconfig` and the plugins from the pod](#access-kubestellar-service-from-the-host-os-by-extracting-the-adminkubeconfig-and-the-plugins-from-the-pod)
+  - [Access **KubeStellar** from another pod in the same `kubestellar` namespace](#access-kubestellar-from-another-pod-in-the-same-kubestellar-namespace)
+  - [Add a new cluster to **KubeStellar** inventory](#add-a-new-cluster-to-kubestellar-inventory)
+
+## Deploy **KubeStellar** in a **Kubernetes** cluster (**Kind** cluster)
+
+Create a **Kind** cluster with the `extraPortMappings` for ports `80` and `443`:
+
+```shell
+kind create cluster --config=- <<EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 80

```

Comment:
Please make the hostPort above 1024.

Reply:
Ok, I have changed the port to 1024 and also removed 80, which is not needed